src/components/ReviewRoutingSlip/ReviewRoutingSlipChequePayment.vue

```diff
@@ -20,7 +20,7 @@
         label="Cheque Date"
         persistent-hint
         hide-details
-        :value="payment.paymentDate ? formatDisplayDate(payment.paymentDate, 'MMMM DD, YYYY') : '-'"
+        :value="payment.paymentDate ? formatDisplayDate(payment.paymentDate.split('T')[0], 'MMMM DD, YYYY') : '-'"
```

Comment:
~~Not really sure how splitting the paymentDate between the date and time is causing it to show the correct date.~~
Oh wait removing time from the date means that when we convert paymentDate into a moment object it doesn't get shot back 8 hours into the previous day ahhh